client/src/components/Workflow/Invocation/Export/ExportToRemoteButton.vue

```diff
@@ -0,0 +1,51 @@
+<script setup>
+import { watch } from "vue";
+import { BButton } from "bootstrap-vue";
+import { FontAwesomeIcon } from "@fortawesome/vue-fontawesome";
+import { useTaskMonitor } from "composables/useTaskMonitor";
+
+const { isRunning, isCompleted, hasFailed, requestHasFailed, waitForTask } = useTaskMonitor();
+
+const props = defineProps({
+    title: { type: String, required: true },
+    taskId: { type: String, default: null },
+});
+
+const emit = defineEmits(["onClick", "onSuccess", "onFailure"]);
+
+watch(
+    () => props.taskId,
+    (newTaskId, oldTaskId) => {
+        if (newTaskId !== oldTaskId) {
+            waitForTask(newTaskId);
+        }
+    }
+);
+
+watch([isCompleted, hasFailed, requestHasFailed], ([newIsCompleted, newHasFailed, newRequestHasFailed]) => {
+    if (newIsCompleted) {
+        emit("onSuccess");
+    }
+    if (newHasFailed || newRequestHasFailed) {
+        emit("onFailure");
+    }
+});
+</script>
+
+<script>
+import { library } from "@fortawesome/fontawesome-svg-core";
+import { faCloudUploadAlt, faSpinner, faCheckCircle, faExclamationCircle } from "@fortawesome/free-solid-svg-icons";
+
+library.add(faCloudUploadAlt, faSpinner, faCheckCircle, faExclamationCircle);
+</script>
+
+<template>
+    <div>
```

Comment:
The group button UI broke because of this parent `div` and can be deleted.
Current (look at the borders)
![image](https://user-images.githubusercontent.com/8046843/196416214-70f8edf2-84a4-481c-ad78-a21d70f44fda.png)
![image](https://user-images.githubusercontent.com/8046843/196415809-079ed399-6d32-46ce-b655-184407776e95.png)

Correct:
![image](https://user-images.githubusercontent.com/8046843/196415951-ea2ea371-79be-4642-a547-5a20c84561eb.png)
![image](https://user-images.githubusercontent.com/8046843/196415922-fda19700-890d-409b-9100-7711c6cde9ea.png)

Reply:
Ohh nice catch! Thank you!